exec/tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {py36,py37,py38}-beam{22,23,24},docs,manifest,check-formatting,lint
+envlist = {py36,py37,py38}-beam{22,23,24,25},docs,manifest,check-formatting,lint
```

Comment:
We only test the latest 3 beam versions

Reply:
if we stop testing on 22 should we also then set the min version for beam to `>2.22`?  I left it in because in my other branch that fixed BQ reader, my first attempt to support beam 2.25 ended up breaking under 2.22, something I would have missed otherwise.

Reply:
I wouldn't set the minimum beam version; we just only explicitly test the latest 3 beam releases to minimize our test matrix a bit (and we're a bit lucky that many of our current users update beam versions frequently).